packages/format-library/src/non-breaking-space/index.js

```diff
@@ -0,0 +1,35 @@
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { insert } from '@wordpress/rich-text';
+import { RichTextShortcut } from '@wordpress/block-editor';
+import { isAppleOS } from '@wordpress/keycodes';
+
+const name = 'core/non-breaking-space';
+const title = __( 'Non breaking space' );
+
+export const nonBreakingSpace = {
+	name,
+	title,
+	tagName: 'span',
+	className: 'nbsp',
+	edit( { value, onChange } ) {
+		function addSpace() {
```

Comment:
Can this be more specific? addNonBreakingSpace?

Reply:
Fixed by 2697e83